roles/sap_general_preconfigure/tasks/RedHat/generic/check-dns-name-resolution.yml

```diff
@@ -1,17 +1,23 @@
 ---
 
-- name: Check dns forwarding settings
+- name: Check forward dns record (A)
   ansible.builtin.shell: test "$(dig {{ sap_general_preconfigure_hostname }}.{{ sap_general_preconfigure_domain }} +short)" = "{{ sap_general_preconfigure_ip }}"
   changed_when: false
-  ignore_errors: true
+ #ignore_errors: true
+  register: dug
```

Comment:
The default in the preconfigure roles is to prepend all variables defined by `register:` with `__<role>_register_`, so please modify the variable name to something similar to `__sap_general_preconfigure_register_dig_a` or `__sap_general_preconfigure_register_dns_resolve_a` .